adaptive_n_back/experiment.js

```diff
@@ -200,6 +234,9 @@ for (var b = 0; b < num_blocks; b++) {
 		};
 		adaptive_n_back_experiment.push(test_block)
 	}
+	if ($.inArray(b,[4,7,15]) != -1) {
```

Comment:
this sort of thing could be called a "dynamic plugin" and as long as they are independent blocks, occurring randomly (or some algorithmically determined frequency and insertion points), it would be possible to add them to any tasks dynamically. It rests on being able to figure out how to do this. If you like the idea I can give this a test tomorrow morning. Adding in the docker infrastructure would be a default variable akin to adding an experiment variable (but wouldn't need to be specified in the config.json unless there is some variance between tasks), but adding dynamically at run time for --run would need to be something like:

      expfactory --run --dynamic-plugin attention-checks